unity-renderer/Assets/DCLPlugins/ECS7/CanvasPainter/UIData/UIDataContainer.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using DCL.Controllers;
+using DCL.ECSComponents;
+using DCL.Models;
+
+namespace DCL.ECS7.UI
+{
+    public class UISceneDataContainer
```

Comment:
I suppose this should implement below's interface?

Reply:
No, the interface is `IUIDataContainer` and it is inherti by UIDataContainer. This class is `UISceneDataContainer`. They are different classes. I have separated it into an additional file to avoid confusion. Sorry!